core/src/main/scala/chisel3/VerificationStatement.scala

```diff
@@ -170,15 +245,14 @@ object assume {
   def _applyWithSourceLine(
```

Comment:
Same comment about binary compatibility

Reply:
Done - due to some compile issues could not use the same name (basically overloading failed) - hence had to rename the new one to be different.